src/pages/Movies.jsx

```diff
@@ -1,89 +1,104 @@
-import { useCallback, useEffect, useMemo } from "react";
+import { useCallback, useEffect } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import List from "../components/Content/List";
-import { fetchAllMovieBookmarks, fetchAllMovies } from "../store/Redux/MovieSlice"
+import { fetchAllMovieBookmarks, fetchAllMovies } from "../store/Redux/MovieSlice"; // Import searchMovies action
```

Comment:
Keep the comment on new line, or remove this comment.

Reply:
resolved